third_party/py/python_configure.bzl

```diff
@@ -139,9 +139,9 @@ def _symlink_genrule_for_dir(repository_ctx,
 def _get_python_bin(repository_ctx):
     """Gets the python bin path."""
     python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH)
```

Comment:
Why not ` python_bin = repository_ctx.os.environ.get(_PYTHON_BIN_PATH, 'python')`?

Reply:
Good idea! Updated!